pkg/infra/tower.go

```diff
@@ -98,11 +102,62 @@ func (t *Tower) reshuffle(numOfFloors int) {
 }
 
 func (t *Tower) hpDecay() {
-	// TODO: can add a parameter
 	for _, agent := range t.agents {
-		newHP := agent.hp - 20
+		newHP := 0
+		switch {
+		// Strong Level
```

Comment:
I think hardcoding 0.63 is fine as we don't intend to parameterize further down the line. However, similarly to my comment on `basegent.go` if there could be better documentation for the formula/reasoning for `newHP` 
I know we won't be maintaining this codebase for very long so it's less for maintenance sake and more about clarity, IMO if at a later point we want to change this formula, we shouldn't have to wonder why these numbers were picked in the first place.

Reply:
Documentation is definitely needed you are right. I will begin to write the documentation when I am done with all requests. 
@woonmoon Do you think I should add more comments directly in the text in addition of the documentation? Or is the documentation sufficient to explain modeling+hard coded number

Reply:
Hmm okay, happy to keep the magic numbers then. But the meanings need to be documented in the code (and probably elsewhere too).